apps/rush-lib/src/logic/operations/OperationExecutionManager.ts

```diff
@@ -95,10 +95,24 @@ export class OperationExecutionManager {
       if (parallelism === 'max') {
         this._parallelism = numberOfCores;
       } else {
-        const parallelismInt: number = parseInt(parallelism, 10);
+        const parsed: number = parseInt(parallelism, 10);
 
-        if (isNaN(parallelismInt)) {
-          throw new Error(`Invalid parallelism value of '${parallelism}', expected a number or 'max'`);
+        if (isNaN(parsed)) {
+          throw new Error(
+            `Invalid parallelism value of '${parallelism}', expected a number, a percentage, or 'max'`
+          );
+        }
```

Comment:
This will always throw if someone specifies a percent.

Reply:
OK, should be fixed now.